federation-js/src/composition/validate/sdl/__tests__/uniqueTypeNamesWithFields.test.ts

```diff
@@ -509,6 +579,16 @@ describe('UniqueTypeNamesWithFields', () => {
       expect(errors[0]).toMatchInlineSnapshot(`
         Object {
           "code": "VALUE_TYPE_NO_ENTITY",
+          "locations": Array [
```

Comment:
Here's another case where it's hard to get specific with the location. Do they need a key somewhere or do they need to delete a key somewhere else? We can't know. I think these locations are fine given that.